packages/pwa-buildpack/lib/WebpackTools/PWADevServer.js

```diff
@@ -170,51 +167,12 @@ const PWADevServer = {
                 oldBefore(app, server);
                 let middleware;
 
-                const gatheringQueryTabs = new Promise((resolve, reject) => {
-                    const { compiler } = server.middleware.context;
-
-                    compiler.hooks.done.tap(
-                        'PWADevServer',
-                        async ({ compilation }) => {
-                            const queryFilePaths = [];
-                            for (const filename of compilation.fileDependencies) {
-                                if (filename.endsWith('.graphql')) {
-                                    queryFilePaths.push(filename);
-                                }
-                            }
-                            try {
-                                resolve(
-                                    await Promise.all(
-                                        queryFilePaths.map(async queryFile => {
-                                            const query = await readFile(
-                                                queryFile,
-                                                'utf8'
-                                            );
-                                            const name = path.relative(
-                                                context,
-                                                queryFile
-                                            );
-                                            return {
-                                                endpoint,
-                                                name,
-                                                query
-                                            };
-                                        })
-                                    )
-                                );
-                            } catch (e) {
-                                reject(e);
-                            }
-                        }
-                    );
-                });
                 /* istanbul ignore next: dummy next() function not testable */
                 const noop = () => {};
                 app.get('/graphiql', async (req, res) => {
                     if (!middleware) {
                         middleware = playgroundMiddleware({
-                            endpoint,
-                            tabs: await gatheringQueryTabs
+                            endpoint
```

Comment:
This can be discussed; we don't have any `.graphql` files anymore, so I didn't know what to render here, so I just removed the tabs entirely. I'm sure with some effort, we could generate tabs from our new pattern, but didn't seem worth it in this scope.